tools/GraphClust/CollectResults/evaluation.py

```diff
@@ -46,3 +47,31 @@ def sh(script):
     toWrite += listOfClasses[i] + "\t" + listOfClusters[i] + '\n'
 with open("RESULTS/fullTab.tabular", "w") as full:
     full.write(toWrite)
+
+
+listOfClasses = []
+listOfClusters = []
+pattern = re.compile("^RF.*$")
+
+with open("RESULTS/fullTab.tabular", 'r') as f:
```

Comment:
Why are you opening the same file twice? Could you not put the if condition into with and check the first line with your regex?